client/me/gravatar-updater/index.jsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import request from 'superagent';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+import FormLabel from 'components/forms/form-label';
+import { getCurrentUser } from 'state/current-user/selectors';
+import Gravatar from 'components/gravatar';
+import * as OAuthToken from 'lib/oauth-token';
+
+export class GravatarUpdater extends React.Component {
```

Comment:
if we `import React, { Component } from 'react'` this line becomes:
`export class GravatarUpdater extends Component`